ci-operator/jobs/openshift/osde2e/openshift-osde2e-main-periodics.yaml

```diff
@@ -2092,6 +2092,8 @@ periodics:
         value: us-east-1
       - name: CONFIGS
         value: aws,stage,addon-suite
+      - name: HIBERNATE_AFTER_USE
+        value: "false"
```

Comment:
this is to prevent [issues with clusters stuck in resuming state](https://issues.redhat.com/browse/OHSS-8873) in a future